tool/gravity/cli/commands.go

```diff
@@ -495,19 +542,6 @@ type UpgradePlanDisplayCmd struct {
 	Output *constants.Format
 }
 
-// RollbackCmd rolls back the specified operation plan phase
-type RollbackCmd struct {
```

Comment:
I've removed the dedicated `gravity rollback` as all plan handling is now in `gravity plan`.

Reply:
Need to update the docs too then.